modules/client/test/integration/constants.ts

```diff
@@ -37,6 +37,18 @@ const ipfsEndpoints = {
 
 const grapqhlEndpoints = {
   working: [
+    {
+      url:
+        "https://httpstat.us/504?sleep=500",
+    },
+    {
+      url:
+        "https://httpstat.us/504?sleep=500",
+    },
+    {
+      url:
+        "https://httpstat.us/504?sleep=500",
+    },
```

Comment:
Are these "working" endpoints?

Reply:
Nope, these are endpoints simulating a 504 error, to test that the client shift is actually happening